rpm/src/main/java/org/eclipse/packager/rpm/Rpms.java

```diff
@@ -72,9 +74,10 @@ public static String dumpValue(final Object value) {
     }
 
     static void writeByteBuffer(final OutputStream stream, final ByteBuffer dataStore) throws IOException {
-        final WritableByteChannel c = Channels.newChannel(stream);
-        while (dataStore.hasRemaining()) {
-            c.write(dataStore);
+        try (WritableByteChannel c = Channels.newChannel(stream)) {
```

Comment:
Does closing the channel also close the stream? I guess if that's the case, then there should be a note on the method explaining that.

Not that I ever did a good just on documenting things :rofl:

Reply:
Yes, it will close the underlying stream, so I added javadoc.